src/test/java/edu/njit/jerse/daikonplusplus/PipelineWithRealLlmIT.java

```diff
@@ -0,0 +1,157 @@
+package edu.njit.jerse.daikonplusplus;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assumptions.assumeTrue;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.nio.file.*;
+import java.util.List;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Timeout;
+import org.junit.jupiter.api.io.TempDir;
+
+/**
```

Comment:
Make sure that this test is disabled by default when you run `./gradlew test`. Then, set up Github Actions CI that runs `./gradlew test` to make sure that the smoke and regression tests are always passing.